src/components/DateTimePicker/DateTimePicker.tsx

```diff
@@ -105,49 +115,85 @@ const DateTimePicker = (props: Props) => {
     todayButton,
     startDate,
     withPortal,
+    isValid,
+    isInvalid,
+    invalidFeedback,
+    validFeedback,
   } = props
 
   return (
-    <InputGroup className={className}>
-      <InputGroup.Prepend>
-        <InputGroup.Text>
-          <Icon icon="calendar" />
-        </InputGroup.Text>
-      </InputGroup.Prepend>
-      <DatePicker
-        className="form-control"
-        dateFormat={dateFormat}
-        dateFormatCalendar={dateFormatCalendar}
-        disabled={disabled}
-        dropdownMode={dropdownMode}
-        endDate={endDate}
-        excludeDates={excludeDates}
-        includeDates={includeDates}
-        inline={inline}
-        locale={locale}
-        maxDate={maxDate}
-        maxTime={maxTime}
-        minDate={minDate}
-        minTime={minTime}
-        monthsShown={monthsShown}
-        onChange={onChange}
-        selected={selected}
-        selectsEnd={selectsEnd}
-        selectsStart={selectsStart}
-        showMonthDropdown={showMonthDropdown}
-        showTimeSelect={showTimeSelect}
-        showYearDropdown={showYearDropdown}
-        showTimeSelectOnly={showTimeSelectOnly}
-        timeFormat={timeFormat}
-        timeIntervals={timeIntervals}
-        timeCaption={timeCaption}
-        todayButton={todayButton}
-        startDate={startDate}
-        withPortal={withPortal}
-      >
-        {children}
-      </DatePicker>
-    </InputGroup>
+    <div>
+      <InputGroup className={className}>
+        <InputGroup.Prepend>
+          <InputGroup.Text>
+            <Icon icon="calendar" />
+          </InputGroup.Text>
+        </InputGroup.Prepend>
+        <DatePicker
+          className={`form-control ${
+            isInvalid ? 'invalid-date-picker' : isValid ? 'valid-date-picker' : ''
+          }`}
+          dateFormat={dateFormat}
+          dateFormatCalendar={dateFormatCalendar}
+          disabled={disabled}
+          dropdownMode={dropdownMode}
+          endDate={endDate}
+          excludeDates={excludeDates}
+          includeDates={includeDates}
+          inline={inline}
+          locale={locale}
+          maxDate={maxDate}
+          maxTime={maxTime}
+          minDate={minDate}
+          minTime={minTime}
+          monthsShown={monthsShown}
+          onChange={onChange}
+          selected={selected}
+          selectsEnd={selectsEnd}
+          selectsStart={selectsStart}
+          showMonthDropdown={showMonthDropdown}
+          showTimeSelect={showTimeSelect}
+          showYearDropdown={showYearDropdown}
+          showTimeSelectOnly={showTimeSelectOnly}
+          timeFormat={timeFormat}
+          timeIntervals={timeIntervals}
+          timeCaption={timeCaption}
+          todayButton={todayButton}
+          startDate={startDate}
+          withPortal={withPortal}
+        >
+          {children}
+        </DatePicker>
+      </InputGroup>
+      {isValid && validFeedback && (
```

Comment:
I'm not sure that we want to require `validFeedback` be present to render it as successful.